backend/app/app/models/user.py

```diff
@@ -1,19 +1,14 @@
-from typing import TYPE_CHECKING
-
 from sqlalchemy import Boolean, Column, Integer, String
-from sqlalchemy.orm import relationship
+from sqlalchemy.dialects.postgresql import ENUM
 
 from app.db.base_class import Base
 
-if TYPE_CHECKING:
-    from .item import Item  # noqa: F401
-
 
 class User(Base):
     id = Column(Integer, primary_key=True, index=True)
     full_name = Column(String, index=True)
     email = Column(String, unique=True, index=True, nullable=False)
+    profile_picture = Column(String, default=None, nullable=True)
     hashed_password = Column(String, nullable=False)
     is_active = Column(Boolean(), default=True)
-    is_superuser = Column(Boolean(), default=False)
-    items = relationship("Item", back_populates="owner")
+    type = Column(ENUM("superuser", "student", "professor", "admin", name="user_type"), nullable=False)
```

Comment:
Hmm, this and the alembic revision seem to differ?

Reply:
yeah, alembic auto generates the revisions so idk why it's slightly different